### PR TITLE
Add new CTA for coalesce 2023 to devblog

### PR DESCRIPTION
**Pull request by joellabes**
## What are you changing in this pull request and why?
Adding a "register for Coalesce" CTA to devblog 

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/blog/ctas.yml [hunk 1]
```diff
@@ -10,3 +10,8 @@
   subheader: Check out guides on getting your warehouse set up and connected to dbt Cloud.
   button_text: Learn more
   url: https://docs.getdbt.com/quickstarts
+- name: coalesce_2023_signup
+  header: Join data practitioners worldwide at Coalesce 2023
+  subheader: Kicking off on October 16th online and in person (Sydney, London and San Diego)
```
Comment (mirnawong1) [inline website/blog/ctas.yml]:
```suggestion
  subheader: Kicking off on October 16th, both online and in-person (Sydney, London, and San Diego)
```

**Review by mirnawong1 — APPROVED** [review]
hey @joellabes ! looks great and wanted to let you know that some of the blogs don't have a right toc sidebar ([example here](https://docs-getdbt-com-git-coalesce-cta-blog-dbt-labs.vercel.app/blog/create-dbt-documentation-10x-faster-with-ChatGPT)) so the cta doesn't appear. 

other than that, it looks good to me! 

ps - should we have a banner on the docs site to share info on coalesce? or is this a question meant for marketing?

**Comment by joellabes** [comment]
>should we have a banner on the docs site to share info on coalesce? or is this a question meant for marketing?

Good q - if you have a look at the thread I shared in slack I think jakki would be grateful if you could move that forward!

**Comment by mirnawong1** [comment]
sick, you the best @joellabes ! appreciate the flag!

**Pull request merged**